tests/utils.js

```diff
@@ -173,6 +173,10 @@ define([
         if (opts.plugins) {
             vellum_options.plugins = opts.plugins;
         }
+        if (vellum_options && vellum_options.features &&
+            _.isUndefined(vellum_options.features.disable_popovers)) {
+            vellum_options.features.disable_popovers = true;
```

Comment:
This disables the bubble popovers unless a test explicitly wants them (only a couple of tests at the moment). Anecdotally  I had a test timing out before this (2 s) that went down to < .5s